src/components/Icons/FAIcon.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { fas } from '@fortawesome/free-solid-svg-icons'
+import { far } from '@fortawesome/free-regular-svg-icons'
+import { fab } from '@fortawesome/free-brands-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
It looks like this dependency isn't included in your package.json - can you add it?